internal/pkg/aws/codepipeline/codepipeline.go

```diff
@@ -0,0 +1,56 @@
+// Package codepipeline provides a client to make API requests to Amazon Elastic Container Service.
+package codepipeline
+
+import (
+	"fmt"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	cp "github.com/aws/aws-sdk-go/service/codepipeline"
+)
+
+type codepipelineClient interface {
+	GetPipeline(*cp.GetPipelineInput) (*cp.GetPipelineOutput, error)
+}
+
+// CodePipeline wraps the AWS CodePipeline client.
+type CodePipeline struct {
+	client codepipelineClient
+}
+
+// Pipeline contains information about the pipeline
+// TODO wrap nested resources or just use what the SDK provides?
+type Pipeline struct {
+	Name string `json:"name"`
+	// Stages        []Stage       `json:"stages"`
+	// ArtifactStore ArtifactStore `json:"artifactStore"`
```

Comment:
Can we remove these comments (and the `Stage`, `ArtifactStore` types) since the code is not used

Reply:
See above comment.